mlflow/utils/proto_json_utils.py

```diff
@@ -193,6 +193,20 @@ def default(self, o):  # pylint: disable=E0202
             return super().default(o)
 
 
+def _extract_pandas_input_types_from_schema(schema):
```

Comment:
If the input is of type csv, it has to be a pandas input. Since a pandas schema will always be a ColSpec, I dont think this entire function is necessary.
Simply using `dtypes = dict(zip(schema.input_names(), schema.pandas_types()))` should suffice.

Reply:
Made the change.